ui/app/mixins/searchable.js

```diff
@@ -40,20 +39,16 @@ export default Mixin.create({
   listSearched: computed('fuse', 'searchTerm', function() {
     const { fuse, searchTerm } = this.getProperties('fuse', 'searchTerm');
     if (searchTerm && searchTerm.length) {
-      if (searchTerm.startsWith('/')) {
-        return regexSearch(searchTerm, fuse);
-      }
-      return fuse.search(searchTerm);
```

Comment:
So do you still need fuse as a dependency?

Reply:
It's not used anywhere else. I almost removed it, but I sorta felt like keeping it around in the event that we want it again...but that's exactly what version control is for. So yeah, I'll remove it.